### DISCUSSION TITLE
virus scanner: FileRepPup [PUP] in version 1.2.1

### DISCUSSION THREAD

**Discussion by Andre0be** [rustdesk/rustdesk#4958] [General]
Hello,

I have an Avast virus scanner. When I start the rustdesk version 1.2.1 comes the message: FileRepPup [PUP] 

Please check this. Thx.

**Comment by dinger1986** [comment]
you need to report a false positive, if everyone does it eventually AVs will whitelist

**Comment by it-podeskidzie** [comment]
I have the same situation - WithSecure -> HEUR/APC.Gendit.
Reported to Withsecure for check.

**Comment by Adi-18** [comment]
I have the same situation
AVG  virus scanner -> Win64:PUP-gen [PUP]

I work for security it based business and used to say "don't install unknown, unsecured app". So this product looks not like ideal for productive environment.

**Reply by dinger1986** [reply]
problem is its "greyware" or classed as potentially unwanted by antivirus companies, thats not RustDesk's fault thats the fault of people using it for badness
